TOOLS/ce/src/main/scala/org.opalj.ce/ConfigEntry.scala

```diff
@@ -0,0 +1,58 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+/**
+ * Stores a value inside the structure of the configNode
+ * @param value is the value stored in the entry
+ * @param comment are all the comments associated with the value
+ */
+case class ConfigEntry(value: String, comment: Comment) extends ConfigNode {
+    /**
+     * Formats the entry into HTML code
+     * @param label required if the Entry is part of an object (Writes the key of the K,V Map there instead). Overrides the label property of the Comment object.
+     * @param HTMLHeadline accepts the HTML syntax of the Headline of the value. Can contain $ label and $ brief flags for filling with content
+     * @param HTMLContent accepts the HTML syntax of the content frame for the value. Must contains a $ content flag for correct rendering
+     * @return returns the Config Entry as HTML code
+     */
+    override def toHTML(label: String, HTMLHeadline: String, HTMLContent: String, sorted: Boolean): String = {
+        var HTMLString = ""
```

Comment:
Possibly use a StringBuilder, or, given that there are only two strings ever added, just concatenate them in a single line, avoiding any mutability here

Reply:
Implemented